### PR TITLE
Issue/11724 app settings crash

### LINKED ISSUE #11724: App crash when accessing App Settings
### Description

The app crashes when accessing app settings.

### Steps to Reproduce

1. On the site screen, tap the profile icon in the header bar
2. Tap App Settings
3. Observe the app crashes

### Crash Report

**Sentry Url**: https://sentry.io/share/issue/294df5db39da45ae997c1c81de8475ca/
**User Count**: 71
**Count**: 112
**First Release**: org.wordpress.android-14.6
**First Seen**: 2020-04-21T04:19:24Z
**Last Release**: org.wordpress.android-14.6
**Last Seen**: 2020-04-22T15:57:02Z
**24 Hours**: 92
**30 Days**: 111

### Screenshots
|  |
| - |
|  |

```
ArrayIndexOutOfBoundsException: length=2; index=2
     at android.preference.ListPreference.getEntry(ListPreference.java:229)
     at android.preference.ListPreference.getSummary(ListPreference.java:174)
     at android.preference.Preference.onBindView(Preference.java:651)
     at android.preference.Preference.getView(Preference.java:585)
     at android.preference.PreferenceGroupAdapter.getView(PreferenceGroupAdapter.java:255)
     at android.widget.AbsListView.obtainView(AbsListView.java:2428)
     at android.widget.ListView.makeAndAddView(ListView.java:2071)
     at android.widget.ListView.fillDown(ListView.java:797)
     at android.widget.ListView.fillFromTop(ListView.java:859)
     at android.widget.ListView.layoutChildren(ListView.java:1814)
...
(54 additional frame(s) were not displayed)
```

### Environment

Device Android Galaxy S10 (SM-G973F), Android version 10

### PR DESCRIPTION
**Pull request by renanferrari**
Fixes #11724

The problem was related to the string array `app_theme_entries`. This array is used to present to the user the options they have for the app theme. On Android 8 and below, it has only two options: `light` and `dark`. For Android 9 and above, it has a third option: `default`.

This string array wasn't supposed to be translatable, but it wasn't marked as so at first, causing it to be [erroneously replicated](https://github.com/wordpress-mobile/WordPress-Android/commit/44869769b2ecd5c956ee12e6caf3326a405b8e9d) to the `strings.xml` file for the following languages:

- ar
- de
- en-gb
- es
- es-MX
- es-VE
- fr
- he
- id
- it
- ja
- nl
- pl
- ro
- ru
- sq
- sv
- tr
- zh-CN
- zh-HK
- zh-TW

Since the array was being "translated" in these files, the system was unable to replace that translation with the one for Android 9 and above, causing that preference to only have two options, even when it was supposed to have three, finally resulting in a `ArrayIndexOutOfBoundsException`, since the default option in those cases is the third one, which was missing.

**TL;DR:** The App Settings screen would crash for any user on Android 9 and above that was using one of those 20 languages.

This PR marks the string array `app_theme_entries` as not translatable and remove all the instances where it was erroneously replicated to those `strings.xml` files.

### To test

0. Make sure you're using a device with Android 9 and above and that the selected language is one of those listed above.
1. On the site screen, tap the profile icon in the header bar
2. Tap App Settings
3. The app shouldn't crash

### Notes
- If those translations were added by some automated process, we may need to make sure they won't be added by mistake again.

---

PR submission checklist:

- [x] I have considered adding unit tests where possible.
- [x] I have considered adding accessibility improvements for my changes.
- [x] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

### CODE REVIEW

**Comment by renanferrari** [comment]
@oguzkocer Pinging you here, as requested.

**Comment by loremattei** [comment]
@oguzkocer looks good to me 👍
Thanks for handling this!
